### DISCUSSION TITLE
Call for community Wave apps

### DISCUSSION THREAD

**Discussion by mturoci** [h2oai/wave#1632] [Show and tell]
## Have you built an interesting Wave app and would like the world to know about it?

We would love to see what our community is achieving 🏆. Please do not hesitate and share your creations with the whole community either via:

* 💬 Commenting on this discussion with screenshots and short description of what's the app doing.
* 💻 If you feel like it, you can even make a PR for adding it to our [public showcase of Wave apps](https://github.com/h2oai/wave-apps).

We cannot wait to see all the amazing projects! 🔥🔥

**Comment by junioraze** [comment]
Hello guys,

We here are using wave as a hybrid of BI / DS application. 

Our customer collects data from any source of information to create clippings for their clients. Our application goes from inspecting every news item to creating word clouds and visually appealing plots (thanks wave) about the data.

  * Dashboard customer data x sentiment analyses
![image](https://github.com/user-attachments/assets/10590cd3-81a5-40c0-9168-6af7283ee0c5)


  * Dashboard Social Media analyses:
![image](https://github.com/user-attachments/assets/a3bd5035-0d5a-4a10-80ad-5c84ce79527c)

  * Side panel of social media after select_marks event on the heatmap:
![image](https://github.com/user-attachments/assets/d137b874-2a58-4662-a4e7-6fc3f522a8e9)
ps: side panels are awesome to give a data deep dive feeling to the customer. 

* We implemented all the table events to syncronize table filters to others visualizations. like md tables and plots. Like below:
![image](https://github.com/user-attachments/assets/c123b161-0e4f-4da7-8349-ce63559c86ec)
ps: The rest of the side panel.

* Integration with plotly for color match (wave treemap when?)
![image](https://github.com/user-attachments/assets/a315b8df-36cd-43a6-93c2-8e867848908a)

As a data engineer doing all of this UI components with just python (and a little of JS to fill some things) is just awesome. Hope to see what others developers are doing to. Hugs!

**Reply by mturoci** [reply]
Looks great, thanks for sharing! cc @mtanco @lo5 @geomodular

**Reply by geomodular** [reply]
Amazing!
